components/admin_console/admin_definition.jsx

```diff
@@ -2465,6 +2465,9 @@ const AdminDefinition = {
                         placeholder: t('admin.ldap.groupFilterEx'),
                         placeholder_default: 'E.g.: "(objectClass=group)"',
                         isDisabled: it.stateIsFalse('LdapSettings.EnableSync'),
+                        isHidden: (config, state, license) => {
+                            return license.LDAPGroups !== 'true';
```

Comment:
Could we have used licensedForFeature()?

Reply:
Yes good call.